documentation/terraform.md

```diff
@@ -71,7 +68,7 @@ The equivalent of the Makefile `dev terraform-app-plan` is:
 cd terraform/app
 rm -f .terraform.lock.hcl
 terraform init -reconfigure -backend-config="key=dev/app.tfstate"
-terraform plan -var="paas_sso_passcode=MyPasscode" -var="app_docker_image=dfedigital/teaching-vacancies:dev-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714" -var-file ../workspace-variables/dev.tfvars.json
+terraform plan -var="sso_passcode=MyPasscode" -var="app_docker_image=dfedigital/teaching-vacancies:dev-08406f04dd9eadb7df6fcda5213be880d7df37ed-20201022090714" -var-file ../workspace-variables/dev.tfvars.json
```

Comment:
`sso_passcode` is only for paas. it should be removed from the command.
Please add the `az login` step instead

Reply:
Also remove the passcode from the make commands